### PR TITLE
Add Offline UX Considerations article

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add the article

### CODE REVIEW

**Comment by kaycebasques** [comment]
@tomayac can you also create (and link to) a PR on WebFu that deletes the old content (including all translated pages) and sets up the redirect (of course we should land this first but I just want to note it before I forget)

**Comment by tomayac** [comment]
> @tomayac can you also create (and link to) a PR on WebFu that deletes the old content (including all translated pages) and sets up the redirect (of course we should land this web.dev PR first but I just want to note the related work before I forget)

See https://github.com/google/WebFundamentals/pull/9003.

**Comment by kaycebasques** [comment]
@tomayac are you waiting on a review from @mustafa-x?

**Comment by tomayac** [comment]
> @tomayac are you waiting on a review from @mustafa-x?

I actually was. Else, someone else from the current web.dev team can obviously take over.

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:413]:
```suggestion
- Think how you design for the success, failure, and instability of a network connection.
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:416]:
```suggestion
- Low end devices are commonplace, with limited storage, memory and processing power, and small
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:423]:
```suggestion
- Utilize language, icons, imagery, typography, and color to express ideas to the user collectively.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [reply src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:23]: The difference is that being in a building is a permanent dead zone, whereas traveling through a tunnel is a temporary dead zone.

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:23]:
```suggestion
- Entering into temporary "dead zones" like when traveling on a train and going through a tunnel.
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:22]:
```suggestion
- Entering into permanent "dead zones" such as buildings with walls that block network connections.
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [reply src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:54]: Emojoy is the name of the app.

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:54]:
```suggestion
       alt="The Emojoy emoji messaging app informing the user when a change in state occurs."
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:192]:
```suggestion
       alt="Tailpiece has a visual indicator that shows what sections are ready for offline use."
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [reply src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:192]: Replaced with "visual indicator".

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:246]:
```suggestion
"offline library" or [content index](/content-indexing-api/),
so the user can easily see what they have stored on their phone and what needs to
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [reply src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:248]: Absolutely! Added.

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:276]:
```suggestion
able to transfer experiences can be incredibly jarring to users.
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [reply src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:276]: Weakened the statement.

**Comment by tomayac** [comment]
Thanks, @jpmedley, I've addressed the helpful review remarks. PTAL.

**Comment by kaycebasques** [comment]
Per our [inactive PR review policy](https://web.dev/handbook/pull-requests/#claim-an-inactive-pr) I will continue this review now

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (kaycebasques) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:288]:
```suggestion
### Use simple, concise language
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (kaycebasques) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:309]: This section reinforces my previous suggestion to rephrase that section heading that is using the word `accessible` in a more general way

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (kaycebasques) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:333]: Let's use a [compare component](https://web.dev/handbook/web-dev-components/#compare) here?

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (kaycebasques) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:348]:
```suggestion
example, save or download data would be action-based.
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (kaycebasques) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:354]:
```suggestion
Offline can mean a number of things depending on the context, such as download, export, pin, etc.
For more inspiration checkout the
[Material Design icon set](https://material.io/resources/icons/).
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (kaycebasques) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:397]:
```suggestion
In many regions, low-end devices are commonplace, connectivity is unreliable and, for many users,
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (kaycebasques) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:407]:
```suggestion
Education is key to offline UX as users are unfamiliar with these concepts. Try to create associations
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (kaycebasques) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:410]:
```suggestion
When designing for unstable network connections, remember these guidelines:
```

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (kaycebasques) [inline src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:415]:
```suggestion
- Low-end devices are commonplace, with limited storage, memory and processing power, and small
```

**Review by kaycebasques — CHANGES_REQUESTED** [review]

**Comment by kaycebasques** [comment]
@tomayac PTAL

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [reply src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md:3]: Absolutely.

File: src/site/_data/paths/progressive-web-apps.json
Comment (tomayac) [inline src/site/_data/paths/progressive-web-apps.json:20]:
```suggestion
        "offline-ux-design-guidelines"
```

**Comment by mustafa-x** [comment]
Sorry for not responding sooner, I have been out dealing with a family emergency

File: src/site/_data/paths/progressive-web-apps.json
Comment (tomayac) [reply src/site/_data/paths/progressive-web-apps.json]: Moved.

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [reply src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md]: Added an aside.

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [reply src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md]: Updated to https://material.io/components/snackbars

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [reply src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md]: Addressed by the new aside I think.

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [reply src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md]: In this context I think "accessible" is correct, since the text talks about color deficiencies.

File: src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md
Comment (tomayac) [reply src/site/content/en/progressive-web-apps/offline-ux-considerations/index.md]: Applied, here and above.

**Comment by tomayac** [comment]
> Sorry for not responding sooner, I have been out dealing with a family emergency

No worries, @mustafa-x, I've taken you off the reviewers list. Family is always a priority. @kaycebasques, I have addressed your remarks. PTAL.

File: src/site/content/en/reliable/offline-ux-design-guidelines/index.md
Comment (kaycebasques) [inline src/site/content/en/reliable/offline-ux-design-guidelines/index.md:351]: @tomayac I won't block you on this but you might enjoy using columns to get these to display side-by-side. See https://web.dev/css-marker-pseudo-element/ for an example:

**Review by kaycebasques — APPROVED** [review]
LGTM, thanks @tomayac

**Comment by kaycebasques** [comment]
@tomayac can you create a PR to redirect the old content?

**Comment by tomayac** [comment]
@kaycebasques Tagged you on the WebFu PR, and opened https://github.com/GoogleChrome/web.dev/pull/4142 as a followup to your suggestion in https://github.com/GoogleChrome/web.dev/pull/3863#discussion_r513737612.

**Pull request merged**
